hackerrank/src/Algorithms/Strings/StringConstruction.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Strings.StringConstruction;
+
+public class Result
+{
+    public static int stringConstruction(string s)
+    {
```

Comment:
Няма праверкі параметра!
```suggestion
    public static int stringConstruction(string s)
    {
        if (string.IsNullOrEmpty(s))
            return 0;
```

Reply:
Геніяльны алгарытм! 😃